colabs/artifact_basics/Artifact_Basics.ipynb

```diff
@@ -0,0 +1,350 @@
+      },
+      "source": [
+        "# Update an Artifact"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "e4psfYzGeAVS"
+      },
+      "source": [
+        "You can pass new values to update the `description`, `metadata`, and `alias` of an artifact. Call the `save()` method to update the artifact on the W&B servers. You can update an artifact during or outside a W&B Run.\n",
+        "\n",
+        "\n",
+        "This example changes the `description` of the `my_first_artifact` artifact inside a run:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "16fCvZUZJD29"
+      },
+      "outputs": [],
+      "source": [
+        "run = wandb.init(project=\"artifact-basics\")\n",
+        "artifact = run.use_artifact(artifact_or_name=\"my_first_artifact:latest\")\n",
+        "artifact.description = \"This is an edited description.\"\n",
+        "artifact.save()\n",
+        "run.finish()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "o-rcMsZYh0Ig"
+      },
+      "source": [
+        "# Download an Artifact"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "B2ssTI7LrHS8"
+      },
+      "source": [
+        "To retrieve the path of an Artifact for external use, use the `artifact.download()` method. This will retrieve the directory of an Artifact you select."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "18-7Kd9Oh8cl"
+      },
+      "outputs": [],
+      "source": [
+        "run = wandb.init(project=\"artifact-basics\")\n",
+        "artifact = run.use_artifact(artifact_or_name=\"my_first_artifact:latest\")\n",
+        "# downloads the artifact to a directory on the W&B platform.\n",
+        "datadir = artifact.download()\n",
+        "run.finish()\n",
+        "# prints the current artifact directory\n",
+        "print(u'\\u2500' * 10)\n",
+        "print(\"Data directory located at:\" + datadir)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "sxox0H9grrcx"
+      },
+      "source": [
+        "For more information on ways to customize your Artifact download, see the [Download and Usage guide](https://docs.wandb.ai/guides/artifacts/download-and-use-an-artifact)."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Navigate the Artifacts UI"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "You can also manage your Artifacts via the W&B platform. This can give you insight into your model's performance or database versioning. To navigate to the relevant infomation, click on any link to a run (these can be found in any of the code samples), then click on the **Artifacts** tab."
```

Comment:
lets be more specific here and give them a link they can navigate to. We should also add a gif so users now how to get to the artifacts page as it is not always obvious. You can also call out that when you log an artifact, the SDK will print the url to access it (this is probably the best way for a user to access it)
```suggestion
        "You can also manage your Artifacts via the W&B platform. This can give you insight into your model's performance or dataset versioning. To navigate to the relevant information, click on any link to a run (these can be found in any of the code samples), then click on the **Artifacts** tab."
```

(these can be found in any of the code samples)
^ we should replace this with an actual link to a run in a public project - it's not very helpful to tell a user to go find a link on a different page on their own. You can also refer to a specific snippet on this page but it has to be clear how to find the link to the run from that snippet